docs/usage/help.rst.inc

```diff
@@ -18,12 +18,12 @@ unwanted ones. Patterns can be used
 - for ``PATH`` arguments that explicitly support them.
 
 Borg always stores all file paths normalized and relative to the
-current recursion root. The recursion root is also named ``PATH`` in
-Borg commands like `borg create` that do a file discovery, so do not
-confuse the root with the ``PATH`` argument of e.g. `borg extract`.
+first directory of the current recursion root. The recursion root is also named
+``PATH`` in Borg commands like `borg create` that do a file discovery, so do
+not confuse the root with the ``PATH`` argument of e.g. `borg extract`.
 
 Starting with Borg 1.2, paths that are matched against patterns always
-appear relative. If you give ``/absolute/`` as root, the paths going
+appear relative. If you give ``/absolute/path/`` as root, the paths going
 into the matcher will start with ``absolute/``.
```

Comment:
I still find the whole discussion of relative paths pretty confusing.  For the root `/absolute/path/`, the first directory is `/absolute`, and if things were "relative" to that first directory, then the paths would *not* start with `absolute`.  I think saying that the paths are "relative" to anything is misleading.  Instead, one could say that the paths that appear *include* the path given as the recursion root, but with any leading slash and leading `../` removed.

Reply:
Yeah, that would be even better!

Reply:
@jdchristensen, I agree. Does this sound more correct?

```
Borg always stores all file paths normalized, with the first part equal to the
recursion root, but (since 1.2) with any leading / or ../ removed. If you give
``/absolute/path`` as root, the paths going into the matcher will start with
``absolute/path``. If you wanted to exclude e.g.
``/absolute/path/logs/*.log``, the pattern ``logs/*.log`` would not match; you
must use ``absolute/path/logs/*.log`` or ``**/logs/*.log`` (using ``sh``
patterns). Since the leading path separator is stripped in patterns,
``/absolute/path/logs/*.log`` will also match (and is clearest).

The recursion root is also named ``PATH`` in Borg commands like `borg create`
that do a file discovery, so do not confuse the root with the ``PATH``
argument of e.g. `borg extract`.

```

Committed but not yet PR'ed at https://github.com/borgbackup/borg/compare/master...jefft:borg:docfix-6231

Reply:
I like the new version.  It's good that it discusses the removal of leading things in the same paragraph and that it gives more examples.  I'm not sure about the sentence that suggests that including the leading / is the clearest, though.

Reply:
This whole business of absolute paths pretending to be relative is weird to me. This makes more sense:
```
R /opt/app
- /opt/app/logs/*.log
```
than:
```
R /opt/app
- opt/app/logs/*.log
```
So I think it's at least worth mentioning the first possibility. I've updated the PR again.